### PR TITLE
Update German

### PR DESCRIPTION
**Pull request by Justman10000**

### CODE REVIEW

File: src/i18n/de.json [hunk 1]
```diff
@@ -11,10 +11,10 @@
     "ui.account.intro1": "Willkommen zum Contao Manager, einem universellen Werkzeug, um das Contao Open Source CMS zu installieren, zu konfigurieren und zu warten. Wenn du den Manager zum ersten Mal einsetzen, {readTheManualToGetStarted}.",
     "ui.account.introGetStarted": "{readTheManual} bevor du beginnst",
     "ui.account.introManual": "lies die Dokumentation",
-    "ui.account.intro2": "Sollten Probleme auftreten, dann prüfe {ourGithubIssues}. Falls es für dein Anliegen noch kein Ticket gibt, kannst du gern ein Neues erstellen.",
+    "ui.account.intro2": "Sollten Probleme auftreten, dann prüfe {ourGithubIssues}. Falls für dein Anliegen noch kein Ticket existiert, kannst du gern ein Neues erstellen.",
```
Comment (fritzmg) [inline src/i18n/de.json:14]: This change seems unnecessary.

File: src/i18n/de.json [hunk 2]
```diff
@@ -158,7 +158,7 @@
     "ui.task.error": "Ein Hintergrund-Prozess wurde unerwartet beendet. Bitte prüfe die Konsolen-Ausgabe.",
     "ui.task.failed": "Der Contao Manager konnte den Hintergrund-Prozess nicht starten.",
     "ui.task.failedDescription1": "Die Ausführung der Aufgabe ist fehlgeschlagen.",
-    "ui.task.failedDescription2": "Sollte dies wiederholt passieren, wird dein Server möglicherweise nicht unterstützt.",
+    "ui.task.failedDescription2": "Sollte dies wiederholt geschehen, wird dein Server möglicherweise nicht unterstützt.",
```
Comment (fritzmg) [inline src/i18n/de.json:161]: This change seems unecessary.

**Review by fritzmg — CHANGES_REQUESTED** [review]
Translations are managed on Transifex: https://www.transifex.com/contao/contao-manager/

You can request being part of the translation team there.

**Comment by aschempp** [comment]
Thank you @Justman10000. As @fritzmg pointed out, translations are managed on Transifex. I like your improved wording and have adopted everything, it will be included in the next release.

**Pull request closed without merging**